### ISSUE TITLE
ERROR: The extracted extension ('mpeg4') is unusual and will be skipped for safety reasons.

### ISSUE THREAD

**Issue by ryan-serpico** [yt-dlp/yt-dlp#10365]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

United States

### Provide a description that is worded well enough to be understood

Prior to `yt-dlp 2024.07.01`, I was able to download videos from the Town of Colonie, New York. 

[Here's an example of a meeting I used to be able to download](https://colonieny.iqm2.com/Citizens/SplitView.aspx?DVR=True&Mode=Video&MeetingID=1623).

Now I'm receiving this error:

```
ERROR: The extracted extension ('mpeg4') is unusual and will be skipped for safety reasons. If you believe this is an error, please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
```

Any way around this? Can mpeg4 be allowlisted?

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
main-1  | [debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8 (No ANSI), error utf-8 (No ANSI), screen utf-8 (No ANSI)
main-1  | [debug] yt-dlp version stable@2024.07.02 from yt-dlp/yt-dlp [93d33cb29] (pip) API
main-1  | [debug] params: {'http_headers': {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:127.0) Gecko/20100101 Firefox/127.0', 'Accept': '*/*', 'Accept-Language': 'en-US,en;q=0.5', 'Sec-Fetch-Mode': 'cors', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Dnt': '1', 'Connection': 'keep-alive', 'Sec-Fetch-Dest': 'empty', 'Sec-Fetch-Site': 'same-site'}, 'outtmpl': '/code/src/data/audios/colonie/2024_07_04_colonie_zoning_board_of_appeals_zba_webcast_jul_3_2024.%(ext)s', 'verbose': True, 'postprocessors': [{'key': 'FFmpegExtractAudio', 'preferredcodec': 'mp3', 'preferredquality': '192'}], 'compat_opts': set()}
main-1  | [debug] Python 3.11.9 (CPython aarch64 64bit) - Linux-6.6.22-linuxkit-aarch64-with-glibc2.36 (OpenSSL 3.0.13 30 Jan 2024, glibc 2.36)
main-1  | [debug] exe versions: ffmpeg 5.1.5-0 (setts), ffprobe 5.1.5-0
main-1  | [debug] Optional libraries: Cryptodome-3.20.0, brotli-1.1.0, certifi-2024.07.04, mutagen-1.47.0, requests-2.32.3, sqlite3-3.40.1, urllib3-2.2.2, websockets-12.0
main-1  | [debug] Proxy map: {}
main-1  | [debug] Request Handlers: urllib, requests, websockets
main-1  | [debug] Loaded 1831 extractors
main-1  | [generic] Extracting URL: https://colonieny.iqm2.com/Citizens/VideoMain.aspx?MeetingID=1623
main-1  | [generic] VideoMain: Downloading webpage
main-1  | [redirect] Following redirect to https://colonieny.iqm2.com/Citizens/SplitView.aspx?DVR=True&Mode=Video&MeetingID=1623
main-1  | [generic] Extracting URL: https://colonieny.iqm2.com/Citizens/SplitView.aspx?DVR=True&Mode=Video&MeetingID=1623
main-1  | [generic] SplitView: Downloading webpage
main-1  | WARNING: [generic] Falling back on generic information extractor
main-1  | [generic] SplitView: Extracting information
main-1  | [debug] Looking for embeds
main-1  | [debug] Identified a JW Player JS loader
main-1  | [download] Downloading playlist: Video Outline - Colonie, NY
main-1  | [generic] Playlist Video Outline - Colonie, NY: Downloading 3 items of 3
main-1  | [download] Downloading item 1 of 3
main-1  | [debug] The information of all playlist entries will be held in memory
main-1  | [debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
main-1  | [debug] Default format spec: bestvideo*+bestaudio/best
main-1  | [info] 1548_720: Downloading 1 format(s): 0
main-1  | [debug] Invoking http downloader on "https://MediaHTTP.IQM2.com/ColonieNY/1548_720.mp4"
main-1  | [download] Destination: /code/src/data/audios/colonie/2024_07_04_colonie_zoning_board_of_appeals_zba_webcast_jul_3_2024.mp4
[download] 100% of    1.13GiB in 00:02:20 at 8.21MiB/s   
main-1  | [debug] ffmpeg command line: ffprobe -show_streams file:/code/src/data/audios/colonie/2024_07_04_colonie_zoning_board_of_appeals_zba_webcast_jul_3_2024.mp4
main-1  | [ExtractAudio] Destination: /code/src/data/audios/colonie/2024_07_04_colonie_zoning_board_of_appeals_zba_webcast_jul_3_2024.mp3
main-1  | [debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i file:/code/src/data/audios/colonie/2024_07_04_colonie_zoning_board_of_appeals_zba_webcast_jul_3_2024.mp4 -vn -acodec libmp3lame -b:a 192.0k -movflags +faststart file:/code/src/data/audios/colonie/2024_07_04_colonie_zoning_board_of_appeals_zba_webcast_jul_3_2024.mp3
main-1  | Deleting original file /code/src/data/audios/colonie/2024_07_04_colonie_zoning_board_of_appeals_zba_webcast_jul_3_2024.mp4 (pass -k to keep)
main-1  | [download] Downloading item 2 of 3
main-1  | [debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
main-1  | [debug] Default format spec: bestvideo*+bestaudio/best
main-1  | [info] 1548_480: Downloading 1 format(s): 0
main-1  | [debug] Invoking http downloader on "https://MediaHTTP.IQM2.com/ColonieNY/1548_480.mp4"
main-1  | [download] Destination: /code/src/data/audios/colonie/2024_07_04_colonie_zoning_board_of_appeals_zba_webcast_jul_3_2024.mp4
[download] 100% of  374.90MiB in 00:00:45 at 8.22MiB/s      
main-1  | [debug] ffmpeg command line: ffprobe -show_streams file:/code/src/data/audios/colonie/2024_07_04_colonie_zoning_board_of_appeals_zba_webcast_jul_3_2024.mp4
main-1  | [ExtractAudio] Destination: /code/src/data/audios/colonie/2024_07_04_colonie_zoning_board_of_appeals_zba_webcast_jul_3_2024.mp3
main-1  | [debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i file:/code/src/data/audios/colonie/2024_07_04_colonie_zoning_board_of_appeals_zba_webcast_jul_3_2024.mp4 -vn -acodec libmp3lame -b:a 192.0k -movflags +faststart file:/code/src/data/audios/colonie/2024_07_04_colonie_zoning_board_of_appeals_zba_webcast_jul_3_2024.mp3
main-1  | Deleting original file /code/src/data/audios/colonie/2024_07_04_colonie_zoning_board_of_appeals_zba_webcast_jul_3_2024.mp4 (pass -k to keep)
main-1  | [download] Downloading item 3 of 3
main-1  | [debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
main-1  | [debug] Default format spec: bestvideo*+bestaudio/best
main-1  | [info] 1548: Downloading 1 format(s): 0
main-1  | ERROR: The extracted extension ('mpeg4') is unusual and will be skipped for safety reasons. If you believe this is an error, please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
main-1  | Traceback (most recent call last):
main-1  |   File "/usr/local/lib/python3.11/site-packages/yt_dlp/YoutubeDL.py", line 179, in wrapper
main-1  |     return func(self, *args, **kwargs)
main-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
main-1  |   File "/usr/local/lib/python3.11/site-packages/yt_dlp/YoutubeDL.py", line 3475, in process_info
main-1  |     dl_filename = existing_video_file(full_filename, temp_filename)
main-1  |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
main-1  |   File "/usr/local/lib/python3.11/site-packages/yt_dlp/YoutubeDL.py", line 3367, in existing_video_file
main-1  |     file = self.existing_file(itertools.chain(*zip(map(converted, filepaths), filepaths)),
main-1  |                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
main-1  |   File "/usr/local/lib/python3.11/site-packages/yt_dlp/YoutubeDL.py", line 3366, in <lambda>
main-1  |     converted = lambda file: replace_extension(file, self.params.get('final_ext') or ext, ext)
main-1  |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
main-1  |   File "/usr/local/lib/python3.11/site-packages/yt_dlp/utils/_utils.py", line 2097, in _change_extension
main-1  |     return f'{filename}.{_UnsafeExtensionError.sanitize_extension(ext)}'
main-1  |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
main-1  |   File "/usr/local/lib/python3.11/site-packages/yt_dlp/utils/_utils.py", line 5131, in sanitize_extension
main-1  |     raise cls(extension)
main-1  | yt_dlp.utils._UnsafeExtensionError: unsafe file extension: 'mpeg4'
main-1  | 
main-1  | 01:56:25 PM CT - hf_download_and_transcribe.py - ERROR - Error downloading audio from https://colonieny.iqm2.com/Citizens/VideoMain.aspx?MeetingID=1623: ERROR: The extracted extension ('mpeg4') is unusual and will be skipped for safety reasons. If you believe this is an error, please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
```

**Comment by bashonly** [comment]
Regression in 5ce582448ececb8d9c30c8c31f58330090ced03a

Will be fixed by #10362

**Comment by bashonly** [comment]
update to nightly: `python3 -m pip install -U --pre "yt-dlp[default]"`
